models/comment.rb

```diff
@@ -108,6 +108,7 @@ def to_hash(params={})
                  .merge("votes" => votes.slice(*%w[count up_count down_count point]))
                  .merge("abuse_flaggers" => abuse_flaggers)
                  .merge("type" => "comment")
+                 .merge("child_comments_count" => Comment.where({"parent_id" => _id}).count())
```

Comment:
@wajeeha-khalid Yes, that sounds quite reasonable. For existing items we can set the count lazily – on load if the field does not exist we calculate and set it.